packages/core/admin/admin/src/components/PageHelpers.tsx

```diff
@@ -54,7 +54,7 @@ const Error = (props: ErrorProps) => {
   const { formatMessage } = useIntl();
 
   return (
-    <Main height="100%">
+    <Main height="100vh">
```

Comment:
I'm not sure this is a good idea, what happens if we render this in under a header? Won't we have a _huge_ scroll space? 👀  would it be better to edit your page container to handle this sizing instead? 

Bottom line, it's gonna be used in lots of places, it's better to make it mouldable to everywhere as opposed to a statement like `100vh`. Thoughts?

Reply:
Yes I hesitated here to do this vs what you suggested. Meant to highlight that when I opened the PR. I think because I saw `Loading` with 100vh so I think I assumed there was a maybe a mistake here and these were meant to match, but what you are saying makes sense, and I guess Loading isn't concerned with a header etc.... I'll revert and handle it on our side.

Reply:
Maybe Loading should be though, and I didn't take that into account 🤔

Reply:
I think Loading is correct to have a 100vh, or at least with 100% like the others it causes issues elsewhere in the application outside the CM.